automation/src/test/scala/org/broadinstitute/dsde/firecloud/fixture/LibraryData.scala

```diff
@@ -37,7 +37,7 @@ object LibraryData {
      "library:technology"->Seq("is an optional","array attribute"),
      "_discoverableByGroups"->Seq("all_broad_users"))
 
-   val metadataORSP = metadataBasic ++ Map(
+   val metadataORSP: Map[String, Any] = metadataBasic ++ Map(
```

Comment:
why this one but not `metadataBasic` too? (though i don't think you need either)